fda-model-test/unittests/AlternativeTest.cs

```diff
@@ -23,8 +23,8 @@ public class AlternativeTest
         static int id = 1;
 
         [Theory]
-        [InlineData(208213.8061,50,.0275,2023,2072,1000)]
-        [InlineData(239260.1814, 50, .0275, 2023, 2050, 1000)]
+        [InlineData(204153.848,50,.0275,2023,2072,1000)]
```

Comment:
was my math wrong?

Reply:
I planned to put my calculations in a public place to show where these estimates came from. Could you share your calculations so that we can document this?

Reply:
This is in part for transparency, and in part to support the internal review/certification effort